### ISSUE TITLE
Import concepts from certlint/crt.sh service for testing X.509 certificates?

### ISSUE THREAD

**Issue by bknowles** [drwetter/testssl.sh#303]
Folks,

Got this in the most recent Fiesty Duck TLS news from the folks at SSLlabs:

> Peter Bowen from Amazon developed the tool certlint to check for several common errors in X.509 certificates. The source code is available on Github and the tool has been integrated into the crt.sh service, a web interface that provides access to the Certificate Transparency logs. Certlint identified several hundred errors and several thousand warnings in valid certificates.

Don't know if this would be useful to you, but thought I'd point it out and let you take a look at it to see if there was anything of interest.

**Comment by drwetter** [comment]
thx for the hint. Need to look closer at it (https://github.com/awslabs/certlint)

First impression was "Sounds good. But it's a different project and a different language"

**Comment by bknowles** [comment]
Yeah, it is a different project and a different language, so I'm sure it would be more work to import any of that into this project.

However, I still think it might be useful to take a look at what they've got and see if there is anything worth "borrowing" from them.

Thanks!
